### PR TITLE
Contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by umair-rao**
I have selected template No. 3 and added the contact form as per template and instrctions, to finish the mobile version.

### CODE REVIEW

**Comment by umair-rao** [comment]
Please view project in following inspect dimenions (497x749) as suggested by first reviewer of project.

File: index.html [hunk 1]
```diff
@@ -216,5 +223,33 @@ <h2 id="about-text">About me</h2>
...
+    <footer class="form">
+        <div class="form-description">
+            <h2 id="form-desc">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        <div class="form">
+            <form action="https://formspree.io/f/mlevbzyw" method="post">
+                <input type="text" id="fname" name="fname" maxlength="30" placeholder="    Full name"><br>
+                <input type="email" id="email" name="email" placeholder="   Email adress">
+                <textarea name="message" maxlength="500"id="textarea"> 
+ Enter text here</textarea>
```
Comment (leonardodiasb) [inline index.html]: - [ ] As the project requires, the user should not be able to submit the form if any of the inputs are empty. Kindly make sure that all the inputs are required in order to submit the form.
 - [ ] The `textarea` input should have a placeholder like the other inputs. By adding a text inside the tag, when the page loads, there is already a sample text and the user can accidentally submit the form without writing a text. Kindly remove this text and use it as a placeholder.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @umair-rao, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Correct use of GitHub flow. ✔️
 - No linter errors. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] Your CSS style is very similar to the Figma template, but it would make a lot of difference if you use the main font-family used by the template, which is the 'Inter' font-family. Kindly make sure you import this font-family with all the font-weights used in the template and use it in the paragraphs, headings, titles, buttons, etc.
-Your design:
image
-Figma template:
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by umair-rao** [comment]
Font family imported and added. I have also made form input filed required and added text in placeholder for textrarea of form.

File: index.html (hunk 1 above)
Comment (umair-rao) [reply index.html]: Made form input filed required and added text in placeholder for textrarea of form. Please check.

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @umair-rao 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**